main.ts

```diff
@@ -0,0 +1,223 @@
+import { App, Modal, Plugin, Setting } from 'obsidian';
```

Comment:
This file was moved to `src/main.ts` and should not be added back in this PR please.